src/util/parse-arguments.util.ts

```diff
@@ -0,0 +1,28 @@
+export function parseArgs(argv: string[]): Record<string, string[]> {
```

Comment:
Awesome! :) Probably worth having a test or two? :P 

Also; I think the `argv.slice(2)`-part could be done by the consumer, don't think this function needs to be aware of node-internal shenanigans